tests/conftest.py

```diff
@@ -209,7 +209,6 @@ def xgboost_version(request):
         "1.11.0",
         "1.12",
         "1.12.0",
-        "1.13",
```

Comment:
what's the purpose behind removing this?

Reply:
ah. i see that it's still in DLC ECR repo, but just not listed on the supported versions:

* https://aws.amazon.com/releasenotes/available-deep-learning-containers-images/

i'll add this back in